test/alternative_spec.rb

```diff
@@ -0,0 +1,51 @@
+require_relative "spec_helper.rb"
+
+describe "UpdateAlternatives::Alternative" do
+
+  context "when we get all alternatives names" do
```

Comment:
I had forgotten about it, you have reason, `self.all` its enought. So, I will set as private.